test/fixtures/corrupted.appiumsession

```diff
@@ -0,0 +1,7 @@
+This is not valid JSON!
```

Comment:
i don't see any of these test fixtures being used; should they be committed?

Reply:
How about I move them over to a new folder called "sample-session-files"?